languages/typescript/src/main/kotlin/io/vrap/codegen/languages/typescript/server/ServerRenderer.kt

```diff
@@ -213,17 +216,22 @@ class ServerRenderer @Inject constructor(
     }
 
     private fun joiValidatorImports(moduleName: String): String {
-        return api.allMethods()
+        val bodyTypes = api.allMethods()
                 .filter { it.bodies.isNotEmpty() }
                 .map { it.bodies[0] }
                 .filter { it.type != null }
                 .map { it.type.toVrapType() }
+        val queryTypes = api.allMethods()
```

Comment:
We also have to add imports for more complex joi validation schemas like `enum` query parameters.